### ISSUE TITLE
bloop config files contain relative paths

### ISSUE THREAD

**Issue by jastice** [propensive/fury-old#239]
Running `make` (fresh download) fails for me because bloop can't find the scala jars:
```
Starting the bsp launcher for bloop...
A bloop installation has been detected either in the PATH or $HOME/.bloop
Server was started in a thread, waiting until it's up and running...
Starting the bloop server with 'bloop server'...
Sleeping for 400ms until we run `bloop about` to check server
Sleeping for 400ms until we run `bloop about` to check server
Sleeping for 400ms until we run `bloop about` to check server
Sleeping for 400ms until we run `bloop about` to check server
Sleeping for 400ms until we run `bloop about` to check server
Sleeping for 400ms until we run `bloop about` to check server
Sleeping for 400ms until we run `bloop about` to check server
Sleeping for 400ms until we run `bloop about` to check server
Sleeping for 400ms until we run `bloop about` to check server
Sleeping for 400ms until we run `bloop about` to check server
Sleeping for 400ms until we run `bloop about` to check server
Sleeping for 400ms until we run `bloop about` to check server
Sleeping for 400ms until we run `bloop about` to check server
Sleeping for 400ms until we run `bloop about` to check server
Sleeping for 400ms until we run `bloop about` to check server
Sleeping for 400ms until we run `bloop about` to check server
Sleeping for 400ms until we run `bloop about` to check server
Sleeping for 400ms until we run `bloop about` to check server
Sleeping for 400ms until we run `bloop about` to check server
Sleeping for 400ms until we run `bloop about` to check server
Sleeping for 400ms until we run `bloop about` to check server
Sleeping for 400ms until we run `bloop about` to check server
Sleeping for 400ms until we run `bloop about` to check server
Sleeping for 400ms until we run `bloop about` to check server
Sleeping for 400ms until we run `bloop about` to check server
error: Spawning a bloop server with 'bloop server' failed!
> Check if server is alive or not...
> [W] Scala instance jar /Users/jast/workspace/fury/bootstrapscala/lib/scala-library.jar doesn't exist!
> [W] Scala instance jar /Users/jast/workspace/fury/bootstrapscala/lib/scala-library.jar doesn't exist!
> [W] Scala instance jar /Users/jast/workspace/fury/bootstrapscala/lib/scala-library.jar doesn't exist!
> [W] Scala instance jar /Users/jast/workspace/fury/bootstrapscala/lib/scala-library.jar doesn't exist!
> [W] Scala instance jar /Users/jast/workspace/fury/bootstrapscala/lib/scala-library.jar doesn't exist!
> [W] Scala instance jar /Users/jast/workspace/fury/bootstrapscala/lib/scala-compiler.jar doesn't exist!
> [W] Scala instance jar /Users/jast/workspace/fury/bootstrapscala/lib/scala-library.jar doesn't exist!
> [W] Scala instance jar /Users/jast/workspace/fury/bootstrapscala/lib/scala-reflect.jar doesn't exist!
> [W] Scala instance jar /Users/jast/workspace/fury/bootstrapscala/lib/scala-compiler.jar doesn't exist!
> [W] Scala instance jar /Users/jast/workspace/fury/bootstrapscala/lib/scala-compiler.jar doesn't exist!
> [W] Scala instance jar /Users/jast/workspace/fury/bootstrapscala/lib/scala-compiler.jar doesn't exist!
> [W] Scala instance jar /Users/jast/workspace/fury/bootstrapscala/lib/scala-library.jar doesn't exist!
> [W] Scala instance jar /Users/jast/workspace/fury/bootstrapscala/lib/scala-compiler.jar doesn't exist![E] /Users/jast/workspace/fury/bootstrapscala/lib/scala-library.jar
>
> [W] Scala instance jar /Users/jast/workspace/fury/bootstrapscala/lib/scala-compiler.jar doesn't exist!
> [W] Scala instance jar /Users/jast/workspace/fury/bootstrapscala/lib/scala-reflect.jar doesn't exist!
```

This is because the bloop config json files contain relative paths, but bloop expects only absolute paths. When I manually change the paths to absolute, the build works.

**Comment by propensive** [comment]
Thanks for the report. I'll update the Make build soon to use absolute paths, but the problem isn't apparent if bloop is started by `make` itself, because it starts bloop in its working directory, from which the paths are relative. In your case, I think you (unsurprisingly) already have a `bloop` server running, started from a different directory.

Anyway, I will update the build to generate the `bloop` files during the build.

**Comment by jastice** [comment]
Yup, having a bloop server running would be the reason I think :)

**Comment by propensive** [comment]
Fixed with #244.
